update_compilers/install_nonfree_compilers.sh

```diff
@@ -116,3 +116,7 @@ install_cuda https://developer.nvidia.com/compute/cuda/10.1/Prod/local_installer
 install_cuda https://developer.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.168_418.67_linux.run 10.1.168
 install_cuda https://developer.download.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_418.87.00_linux.run 10.1.243
 install_cuda https://developer.download.nvidia.com/compute/cuda/10.2/Prod/local_installers/cuda_10.2.89_440.33.01_linux.run 10.2.89
+
+##################################
+# Sonar Analyzer
+s3fetch ${S3URL}/sonar/sonar.txz | tar Jxf -
```

Comment:
Looks good! We will eventually need to come up with a `ce_install` approach for this but that's not important right now :) Thanks!